### PR TITLE
Bugfix for PackageInspector

### PR DESCRIPTION
**Pull request by iambmelt**
Fixes a bug with querying package info when apps have an account type in AccountManager

### CODE REVIEW

File: package-inspector/src/main/java/com/microsoft/inspector/MainActivity.java [hunk 1]
```diff
@@ -121,8 +122,14 @@ private void filterAndDisplayPackages(@Nullable final String filterText) {
             @Override
             public void onItemClick(AdapterView<?> adapterView, View view, int position, long l) {
                 try {
+                    String pkgName = packageNames.get(position);
+
+                    if (pkgName.endsWith(ACCOUNT_AFFINITY_SUFFIX)) {
```
Comment (shahzaibj) [inline package-inspector/src/main/java/com/microsoft/inspector/MainActivity.java:127]: Why does it have that suffix? seems sort of a strange suffix

Comment (iambmelt) [reply package-inspector/src/main/java/com/microsoft/inspector/MainActivity.java:127]: Any app that has an account type in AccountManager has one put on there so you can differentiate in the list... I'm totally open to changing it to something else

image

Comment (iambmelt) [reply package-inspector/src/main/java/com/microsoft/inspector/MainActivity.java:127]: image

Comment (shahzaibj) [reply package-inspector/src/main/java/com/microsoft/inspector/MainActivity.java:127]: I see. I think this is fine

**Review by shahzaibj — APPROVED** [review]
LGTM. Thanks @iambmelt

**Review by rpdome — APPROVED** [review]

**Pull request merged**